canonical-web-and-design/ubuntu.com
# Add subscription group and card components

## Done

- Rename subscribe pattern to subscriptions.
- Add subscription group and card components.

## QA

- Visit /advantage?test_backend=true
- You should see group titles and context menu and cards with (dummy) content.


## Issue / Card

Fixes canonical-web-and-design/commercial-squad#95.
Fixes canonical-web-and-design/commercial-squad#96.

## Screenshots

<img width="433" alt="Screen Shot 2021-08-02 at 3 27 13 pm" src="https://user-images.githubusercontent.com/361637/127810241-e46f5d07-2bf4-413b-a1e7-c463fd8ff5d1.png">
